regression/contracts/loop_contracts_do_while/test.desc

```diff
@@ -0,0 +1,9 @@
+KNOWNBUG
+main.c
+--apply-loop-contracts
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+--
+--
+This test checks that loop contracts work correctly on do/while loops.
```

Comment:
I am slightly confused about this commit.  You have a test to check that things work but the code that is added is error logging.  So... does this example work, but not exercise the code you added?

Reply:
do/while doesn't work yet

Reply:
So this is marked as `KNOWNBUG` and it's not even run on the CI yet.

May be I should change it to `CORE` and check for the expected error message on the output.

Reply:
Ah; forgive me, I missed that.  Yeah, `KNOWNBUG` is sufficient.  `CORE` with the error message is the extra classy option!